.github/workflows/docker.yml

```diff
@@ -45,8 +45,6 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
       - name: Run Java tests
         uses: eskatos/gradle-command-action@v1
-        env:
-          versionTag: ${{ steps.prep.outputs.versionTag }}
```

Comment:
I agree with removing this. I'm not sure if we need the `Prepare version name` job at all. I see that it used below in two jobs name, but I wonder if we could just get rid of that as well. Thoughts? cc @barbeau

Reply:
I think so - from looking at the script it seems like the commands that set up the Docker image version tag are separate from the script setting the validator version tag.

Reply:
I went ahead and dropped all the versioning code.